addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -197,6 +197,8 @@ options.registry.gallery = options.Class.extend({
             _.each(cols, function (col) {
                 var $col = $(col);
                 var height = $col.is(':empty') ? 0 : $col.find('img').last().offset().top + $col.find('img').last().height() - self.$target.offset().top;
+                // Neutralize invisible sub-pixel height differences.
```

Comment:
I'm not convinced by this commit, but in any case when I put the same images in 14.0, I have the same bug (the set to previous position moves the entire col). I suppose you can create an other PR that target 14.0, but if this is the expected behavior for images that have different height, isn't the fix about resize the default image ?

Reply:
Extracted to a 14.0 PR: https://github.com/odoo/odoo/pull/119064.
IMHO it has to be fixed there - the default image causing the error is just a random event that could have happened to any user after e.g. manual cropping.